src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewCell.swift

```diff
@@ -95,6 +102,34 @@ extension DynamicCell {
 			configure?(viewController, cell, indexPath)
 		}
 	}
+	
+	static func doubleLabels(
+		text1: String,
```

Comment:
Maybe would be a nice case to use a tuple?

Reply:
After trying around, i'd like to not change it, because for the texts we could use a tuple, but for the accessibilityIdentifer and the accessibilityTraits we cannot because of the default values.
And so we would expect tuples and not-tuples, what would confuse we if i use this element.